Thesis/StepIndexedRelBigStepTypedAnfIlcCorrect.agda

```diff
@@ -250,6 +302,9 @@ mutual
   rrelV3 (σ ⇒ τ) (closure {Γ1} t1 ρ1) (dclosure {ΔΓ} dt ρ dρ) (closure {Γ2} t2 ρ2) n =
       Σ ((Γ1 ≡ Γ2) × (Γ1 ≡ ΔΓ)) λ { (refl , refl) →
         ρ1 ≡ ρ ×
+        ρ1 ⊕ρ dρ ≡ ρ2 ×
```

Comment:
@yurug Here I write that ρ1 ⊕ρ dρ ≡ ρ2. I could (maybe should) be more restrictive and require that dρ is a valid change from ρ1 to ρ2, at the cost of making rrelρ3 part of the recursion. The fundamental lemma could still be adapted easily.